src/halmos/__main__.py

```diff
@@ -749,7 +752,7 @@ def main() -> int:
     if args.statistics:
         print(f'\n[time] total: {main_end - main_start:0.2f}s (build: {main_mid - main_start:0.2f}s, tests: {main_end - main_mid:0.2f}s)')
 
-    if (total_passed + total_failed) == 0:
+    if not funsigs:
```

Comment:
errors running `setUp()` used to result in the misleading `Error: No tests with the prefix...` message

Reply:
actually this is not quite right, `funsigs` may not have been assigned

Reply:
ah you already fixed that! :)